exercises/split-second-stopwatch/canonical-data.json

```diff
@@ -0,0 +1,194 @@
+{
+  "exercise": "split-second-stopwatch",
+  "comments": ["Times are formatted as 'HH:mm:ss'."],
+  "cases": [
```

Comment:
I noticed there are exactly two cases. Could those be merged and unnested?

Reply:
Well, we could, but there will be more test cases. I just didn't want to add the others before getting a confirmation that the structure I was using was fine